include/dpp/cluster_coro_calls.h

```diff
@@ -233,7 +233,7 @@ awaitable<confirmation_callback_t> co_guild_commands_get(snowflake guild_id);
  * @return confirmation returned object on completion
  * \memberof dpp::cluster
  */
-awaitable<confirmation_callback_t> co_interaction_response_create(snowflake interaction_id, const std::string &token, const interaction_response &r);
+[[nodiscard]] async<confirmation_callback_t> co_interaction_response_create(snowflake interaction_id, const std::string &token, const interaction_response &r);
```

Comment:
Would `std::string_view` not be better here or is it not permitted?

Reply:
It would! Sadly cluster_coro_calls.* and cluster_sync_calls.* are auto-generated from dpp::cluster methods, they grab the parameters from them, so that is not really the purview of this PR 😔

Reply:
Ahhhh, gotcha :)